app/Main.hs

```diff
@@ -1,6 +1,17 @@
 module Main where
 
-import Lib
+import Control.Monad (join)
+
+import Options.Applicative
+
+import Cache (cacheBoards)
```

Comment:
might want to consider prefixing your entire app by it's title or something defining